readinglist/resource.py

```diff
@@ -130,7 +130,11 @@ def _extract_filters(self, queryparams):
             if param in self.known_fields:
                 filters.append((param, value, '=='))
             if param == '_since':
-                filters.append((self.modified_field, value, '>='))
+                if isinstance(value, six.integer_types + (float,)):
+                    filters.append((self.modified_field, value, '>='))
```

Comment:
The benefit with constants is avoiding typos in the string. You import and uses variables everywhere in the code.